nat-lab/tests/test_teliod.py

```diff
@@ -30,56 +40,31 @@ async def test_teliod() -> None:
         )
 
         # Let the daemon start
-        await asyncio.sleep(1)
+        while not await is_teliod_running(connection):
```

Comment:
What if the daemon does not start due to some reason ? It could get stuck here.

Reply:
Yeah, that would be an error and in such case the test timeout will kill it anyway and the test will fail - I was told that this is the current way of doing things i such cases

Reply:
Sounds good!